conf/machine/imx8mp-var-dart-hmx1.conf

```diff
@@ -5,6 +5,8 @@ require conf/machine/include/hmx-base.conf
 
 MACHINEOVERRIDES_EXTENDER:imx8mp-var-dart-hmx1 = "imx8mp-var-dart"
 
+MACHINE_FEATURES = "usbgadget usbhost"
```

Comment:
I suggest:
```bitbake
MACHINE_FEATURES="usbgadget usbhost vfat alsa pci bluetooth wifi" 
```
(considering USB based interfaces)

currently, we have:
```
MACHINE_FEATURES="usbgadget usbhost vfat alsa touchscreen screen  pci bluetooth wifi screen rtc qemu-usermode"
```
(rtc qemu-usermode on backfill)

Reply:
our custom board does not have vfat alsa touchscreen screen pci bluetooth wifi screen. not sure what qemu-usermode does. 
if the next variant has it it should be filled in. I am a bit confused about what we currently use.. is it from conf/machine/imx8mp-var-dart.conf?